app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -105,6 +106,21 @@ def self.params_for_create(ems)
     }
   end
 
+  def params_for_clone
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'name',
+          :id         => 'name',
+          :label      => _('Volume Base Name'),
+          :isRequired => true,
+          :helperText => _("Base name of the new cloned volume(s). Cloned Volume names will be prefixed with 'clone-' and suffixed with '-#####' (where ##### is a 5 digit random number)"),
```

Comment:
Minor, but through this interface we can only clone a single volume so it would be clearer to keep the help text singular. (in my earlier review I lazily copy/pasted the text from the [Power Cloud API docs](https://cloud.ibm.com/apidocs/power-cloud#pcloud-v2-volumes-clone-post)).
```suggestion
          :helperText => _("Base name of the new cloned volume. The cloned Volume name will be prefixed with 'clone-' and suffixed with '-#####' (where ##### is a 5 digit random number)"),
```